src/main/diagnostics/connectedToInternet.ts

```diff
@@ -2,19 +2,36 @@ import fetch from 'node-fetch';
 
 import { DiagnosticsCategory, DiagnosticsChecker } from './types';
 
+import Logging from '@/utils/logging';
+
+const console = Logging.diagnostics;
+
+let timeout = 5000;
+
+if (process.env.RD_CONNECTED_TO_INTERNET_TIMEOUT) {
+  timeout = parseInt(process.env.RD_CONNECTED_TO_INTERNET_TIMEOUT);
+}
```

Comment:
Instead of caching the value, I would like to query the environment variable every time we start the timeout.

If/when #3284 is implemented, we will be able to change settings on-the-fly without stopping the app. Querying the variable is cheap, so caching doesn't really matter, but changing settings at runtime will help us with writing automated tests.